src/js/components/Calendar/StyledCalendar.js

```diff
@@ -123,12 +123,7 @@ const rangeRoundStyle = (props) => {
     themeObj =
       props.theme.calendar?.[props.sizeProp]?.range?.round ||
       props.theme.calendar?.medium?.range?.round;
-  return (
-    themeObj && [
-      roundStyle(themeObj, props.responsive, props.theme),
-      'overflow: hidden;',
```

Comment:
why did we remove the overflow?

Reply:
The overflow hidden was preventing the focus indicator from being displayed, and when I removed it I didn't see any negative side effects from not having it